File: shoring/blackjackEngine/src/test/java/com/panopset/tests/blackjackEngine/DefaultResourcesTest.java
package com.panopset.tests.blackjackEngine;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;
import com.panopset.compat.Stringop;
import com.panopset.blackjackEngine.DefaultResources;

public class DefaultResourcesTest {

  @Test
  void test() {
    String rawData = DefaultResources.getDefaultBasicStrategy();
    Assertions.assertEquals(" 11  Dh Dh Dh Dh Dh Dh Dh Dh Dh Hd", Stringop.stringToList(rawData).get(5));
    rawData = DefaultResources.getDefaultCountingSystems();
    Assertions.assertEquals(" 0 +1 +1 +1 +1  0  0  0 -1  0 Hi-Opt I", Stringop.stringToList(rawData).get(5));
  }
}

File: code/shoring/blackjackEngine/src/test/kotlin/com/panopset/tests/blackjackEngine/DefaultResourcesTest.kt
package com.panopset.tests.blackjackEngine

import com.panopset.blackjackEngine.DefaultResources
import com.panopset.compat.Stringop.stringToList
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class DefaultResourcesTest {

    private val defaultResources = DefaultResources()
    @Test
    fun test() {
        var rawData = defaultResources.defaultBasicStrategy
        Assertions.assertEquals(" 11  Dh Dh Dh Dh Dh Dh Dh Dh Dh Hd", stringToList(rawData)[5])
        rawData = defaultResources.defaultCountingSystems
        Assertions.assertEquals(" 0 +1 +1 +1 +1  0  0  0 -1  0 Hi-Opt I", stringToList(rawData)[5])
    }
}
